grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -70,12 +70,13 @@ class OccurrenceTagLib {
             output = fieldName[0..-4].replaceAll("_", " ") + " (range)"
         } else {
 
-            def label = message(code:"facet." + fieldCode, default:fieldName)
+            def label = message(code:"facet." + fieldCode, default:"")
```

Comment:
I realized that the previous default value was preventing to exec the next message lookup and that many messages without "facet." are still there